handlers/dashboard.go

```diff
@@ -156,7 +156,7 @@ func Dashboard(w http.ResponseWriter, r *http.Request) {
 	validatorLimit := getUserPremium(r).MaxValidators
 
 	q := r.URL.Query()
-	queryValidators, err := parseValidatorsFromQueryString(q.Get("validators"), validatorLimit)
+	_, err := parseValidatorsFromQueryString(q.Get("validators"), validatorLimit)
```

Comment:
**Minor:** It seems counterintuitive to call `parseValidatorsFromQueryString` when the return value does not matter.
I guess you do that to get its limit handling i.e. if too many validators are passed you want the `err` to be returned.

While I guess this is fine (even though it includes and unnecessary overhead), you should at least add a comment:
```suggestion
	// Using parseValidatorsFromQueryString to check for validatorLimit
	_, err := parseValidatorsFromQueryString(q.Get("validators"), validatorLimit)
```

Reply:
Yes, that sounds reasonable enough to me. I also do not like it that way, but the function introduces some error handling. In the future when a bigger refactor is considered we can optimize that part. For now the `parseValidatorsFromQueryString` does not add a big overhead. The function is O(n), and with a max of 300 validators check, is not something to consider for performance issues.